examples/with-esm/package.json

```diff
@@ -12,18 +12,18 @@
 		"#rootJson/*": "./src/*.json"
 	},
 	"dependencies": {
-		"@sapphire/discord-utilities": "^2.12.0",
-		"@sapphire/discord.js-utilities": "^5.1.2",
+		"@sapphire/discord-utilities": "pr-434",
```

Comment:
Updating package.json with...pr versions feels wrong, maybe we should just update it to 4.x for most of these (or whatever their next version is)

Reply:
There is a merge and release order so this will change before merge. That's why it's also labeled blocked.